hawkbit-repository/hawkbit-repository-api/src/main/java/org/eclipse/hawkbit/repository/model/ActionProperties.java

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright (c) 2019 Bosch Software Innovations GmbH and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.hawkbit.repository.model;
+
+import java.io.Serializable;
+
+/**
+ * Holds properties for {@link Action}
+ */
+public class ActionProperties implements Serializable {
```

Comment:
Is implementing `Serializable` really required?

Reply:
Yes, because the ActionProperties gets sent along with the DistributedEvent, therefore it needs to be serializable.

Reply:
@schabdo is this resolved?